README.md

```diff
@@ -10,9 +10,19 @@ This repository contains a set of simple sample projects to help you getting sta
 
 ## How the samples are organized
 
-The samples are divided by the category and [Service Fabric programming model][service-fabric-programming-models] that they focus on: Reliable Actors, Reliable Services. Most real applications will include a mixture of the concepts and programming models. Note that the support for reliable stateful service is not there today and will come in near future.
-
-## Actor samples
+The samples are divided by the category and [Service Fabric programming model][service-fabric-programming-models] that they focus on: Reliable Actors, Reliable Services. Most real applications will include a mixture of the concepts and programming models.
+
+### Folder Hierarchy
+* src/ - Source of the application divided by different modules by sub-folders.  
+* &lt;application package folder&gt;/ - Service Fabric Application folder heirarchy. After compilation the executables are placed in code subfolders.  
```

Comment:
Some miscommunication here.. How does the below look like? 
* <Application Name>/ - This constitutes the Service Fabric application package which can be deployed on cluster. After compilation the binaries are placed in the code subfolder.